tools/tests/interaction-manager/actions/test.js

```diff
@@ -1,11 +1,12 @@
+/* global _IM */
 let actionWorking = false;
 const actionName = 'registered-action';
 
 describe('Actions', () => {
     it('Should register an action', () => {
-        interactionManager.actions.register(actionName, () => {actionWorking = true});
+        interactionManager.actions.register(actionName, () => { actionWorking = true; });

```

Comment:
Can it be done like that?

```suggestion
        interactionManager.actions.register(actionName, () => actionWorking = true);
```

Reply:
Yes - changed.